packages/airnode-node/src/api/index.ts

```diff
@@ -78,7 +78,11 @@ function buildOptions(payload: CallApiPayload): adapter.BuildRequestOptions {
   }
 }
 
-async function signWithRequestId(requestId: string, data: string, airnodeWalletPrivateKey: string) {
+async function signWithRequestId(requestId: string, data: string) {
+  const airnodeWalletPrivateKey = getEnvValue('AIRNODE_WALLET_PRIVATE_KEY');
+  if (!airnodeWalletPrivateKey) {
+    throw new Error('Missing Airnode wallet private key in environment variables.');
+  }
   const airnodeWallet = getAirnodeWalletFromPrivateKey(airnodeWalletPrivateKey);
```

Comment:
This whole thing works the same way for both `signWith*` functions. Can it be abstracted into a separate function?